src/convertToObject.js

```diff
@@ -12,7 +12,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const properties = sourceString
+    .split(';')
+    .map(property => property.split(':'))
+    .filter(property => property.length === 2);
+  const addProperty = (styles, [key, value]) => {
+    return {
+      ...styles,
+      [key.trim()]: value.trim(),
+    };
+  };
```

Comment:
You don't use this callback anywhere else, therefore it is better to write it inside reduce call.